curvesim/tools/bonding_curve.py

```diff
@@ -49,49 +55,104 @@ def bonding_curve(pool, *, truncate=0.0005, resolution=1000, plot=False):
     """
 
     if isinstance(pool, CurveMetaPool):
-        combos = [(0, 1)]
+        combos: List[Tuple[int, int]] = [(0, 1)]
```

Comment:
Custom types that would make the module more readable are:

```
IndexPair = Tuple[int, int]
Point = Tuple[float, float]
```

These can be used just internally within the module for readability.

Reply:
Added these as well as a NormalizedPoint = Tuple[int, int] type for integer balances that aren't in human-readable form.